ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -312,7 +312,18 @@ open class PillButtonBar: UIScrollView {
             let button = createButtonWithItem(item)
             buttons.append(button)
             stackView.addArrangedSubview(button)
-            button.accessibilityHint = String(format: "Accessibility.MSPillButtonBar.Hint".localized, index + 1, items.count)
+
+            var shouldAddAccessibilityHint: Bool = true
+            if #available(iOS 14.6, *) {
+                // in case pillbuttonbar is used as .tabbar, adding our own index would be repetitive
+                // However, iOS 14.0 - 14.5 `.tabBar` accessibilityTrait does not read out the index automatically
+                shouldAddAccessibilityHint = !self.accessibilityTraits.contains(.tabBar)
+            }
+
+            if shouldAddAccessibilityHint {
+                button.accessibilityHint = String(format: "Accessibility.MSPillButtonBar.Hint".localized, index + 1, items.count)
```

Comment:
nit: Not sure if it's going to make a practical difference here, but we've been using [localizedStringWithFormat:](https://developer.apple.com/documentation/foundation/nsstring/1497301-localizedstringwithformat) so that the arguments can also be localized if needed.